packages/theme/pages/Category.vue

```diff
@@ -1,221 +1,239 @@
+                  data-testid="product-link"
+                  v-on="$listeners"
+                >
+                  <template v-if="Array.isArray(imageSlotProps.image)">
                     <nuxt-img
-                      v-else
+                      v-for="(picture, key) in imageSlotProps.image.slice(0, 2)"
+                      :key="key"
                       :alt="imageSlotProps.title"
                       :height="imageSlotProps.imageHeight"
-                      :src="imageSlotProps.image"
+                      :src="picture"
                       :width="imageSlotProps.imageWidth"
-                      class="sf-product-card__image lol"
+                      class="sf-product-card__picture"
                     />
-                  </SfButton>
-                </template>
-              </SfProductCard>
-            </transition-group>
-            <transition-group
-              v-else
-              appear
-              class="products__list"
-              name="products__slide"
-              tag="div"
+                  </template>
+                  <nuxt-img
+                    v-else
+                    :alt="imageSlotProps.title"
+                    :height="imageSlotProps.imageHeight"
+                    :src="imageSlotProps.image"
+                    :width="imageSlotProps.imageWidth"
+                    class="sf-product-card__image lol"
+                  />
+                </SfButton>
+              </template>
+            </SfProductCard>
+          </transition-group>
+          <transition-group
+            v-else
+            appear
+            class="products__list"
+            name="products__slide"
+            tag="div"
+          >
+            <SfProductCardHorizontal
+              v-for="(product, i) in products"
+              :key="getSlug(product)"
+              :description="getDescription(product)"
+              :image="getMagentoImage(getProductThumbnailImage(product))"
+              :image-height="imageSizes.productCardHorizontal.height"
+              :image-width="imageSizes.productCardHorizontal.width"
+              :is-in-wishlist="isInWishlist({product})"
+              :is-in-wishlist-icon="isAuthenticated ? '' : ''"
+              :link="
+                localePath(
+                  `/p/${getProductSku(
+                    product
+                  )}${getSlug(product, product.categories[0])}`
+                )
+              "
+              :regular-price="$fc(getPrice(product).regular)"
+              :reviews-count="getTotalReviews(product)"
+              :score-rating="getAverageRating(product)"
+              :special-price="getPrice(product).special && $fc(getPrice(product).special)"
+              :style="{ '--index': i }"
+              :title="getName(product)"
+              :wishlist-icon="isAuthenticated ? '' : ''"
+              class="products__product-card-horizontal"
+              @click:add-to-cart="addItemToCart({ product, quantity: 1 })"
```

Comment:
Missing changes from https://github.com/vuestorefront/magento2/pull/680
```suggestion
              @click:add-to-cart="addItemToCart({ product, quantity: $event })"
```